website/install.md

```diff
@@ -72,3 +72,11 @@ Git Town runs anywhere Git and [Bash](https://www.gnu.org/software/bash/bash.htm
 #### Install autocompletion
 
 * for [Fish shell](http://fishshell.com): `git town install-fish-autocompletion`
+
+
+#### Notifications about new releases
+
+* Subscribe to our
+  [release RSS feed](https://github.com/Originate/git-town/releases.atom)
```

Comment:
how about just "release feed"? We are giving a link to an `.atom` and Atom is an RSS alternative.